pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "docling-core"
-version = "2.48.4"  # DO NOT EDIT, updated automatically
+version = "2.43.0"  # DO NOT EDIT, updated automatically
```

Comment:
This should be reverted; the version gets bumped by our CD pipeline upon release.